tests/hardhat/Tokens/mintAndRedeemTest.ts

```diff
@@ -291,10 +285,8 @@ describe("VToken", function () {
       });
 
       it("fails if comptroller tells it to", async () => {
-        comptroller.redeemAllowed.returns(11);
-        await expect(redeemFresh(vToken, redeemer, redeemTokens, redeemAmount))
-          .to.be.revertedWithCustomError(vToken, "RedeemComptrollerRejection")
-          .withArgs(11);
+        comptroller.preRedeemHook.reverts();
+        await expect(redeemFresh(vToken, redeemer, redeemTokens, redeemAmount)).to.be.reverted;
```

Comment:
keep error check for these reversions?

Reply:
Smock doesn't preserve error reasons, i.e. this fails:
```
comptroller.preRedeemHook.reverts("Ouch!");
await expect(redeemFresh(vToken, redeemer, redeemTokens, redeemAmount)).to.be.revertedWith("Ouch!");
```

but this succeeds:
```
comptroller.preRedeemHook.reverts("Ouch!");
await expect(redeemFresh(vToken, redeemer, redeemTokens, redeemAmount)).to.be.reverted;
```

🤷